Mailer/app/api/Validation/InputValidatorTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Remp\MailerModule\Api\Validation;
+
+use Nette\Http\Response;
+use Nette\Utils\Json;
+use Nette\Utils\JsonException;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Tracy\Debugger;
+use JsonSchema\Validator;
+
+trait InputValidatorTrait
```

Comment:
* Please rename it to `JsonValidationTrait` since it only supports JSON schema validation against JSON request.
* It can be moved one folder up into the `Remp\MailerModule\Api` namespace.